syntax = "proto3";

package FoodBot;

service FoodBotRLRequest
{
//Input NLG
//Output NLG 
	rpc GetRLResponse(EnvornmentInfomration) returns (Policy){}
}

message Policy
{
	int32 policyNumber = 1;
}

message EnvornmentInfomration
{
	repeated int32 formerState = 2;
	repeated int32 currentState = 3;
	double rewardForTheFormer = 4;
	int32 formerAction = 5;
	bool   shouldTerminate = 6;
}